data_path: /home/rise/catkin_ws/src/data
data_dir: catkin_ws/src/push_planners/twc-stable_push_net_ros/models/2024-05-20-0000
model_dir: catkin_ws/src/push_planners/twc-stable_push_net_ros/models


# Inference Configuration
planner:
  gripper_width: 0.08
  # height: 0.02
  height: 0.03
  # height: 0.05
  num_push_directions: 8

  visualize: True
  # visualize: False
  image_type: 'masked_image'

depth_based:
  friction_coefficient: 1

network:
  model_name: "2024-05-20-0000"

hybrid:
  grid_size: 0.05  # unit: m
  dtheta:  5  # unit: deg

# Train Configuration
file_zero_padding_num: 7
base_lr: 0.0001
decay_rate: 0.95
train_l2_regularizer: 0.0005
batch_size: 64
num_epochs: 10
momentum_rate: 0.9
network_threshold: 0.4
# network_threshold: 0.5
network_input: 4000
StablePushNet:
  image_height: 96
  image_width: 96
  batch_size: 64
